world/client.cpp

```diff
@@ -538,6 +538,16 @@ bool Client::HandleNameApprovalPacket(const EQApplicationPacket *app)
 	uchar race = app->pBuffer[64];
 	uchar clas = app->pBuffer[68];
 
+	if (race < 0 || race > 255) {
+		LogInfo("Client::HandleNameApprovalPacket Race was less then zero or over 255");
+		return false;
+	}
+
+	if (clas < 0 || clas > 255) {
+		LogInfo("Client::HandleNameApprovalPacket Class was less then zero or over 255");
```

Comment:
Drop `Client::HandleNameApprovalPacket` as it comes with logging for free now

Reply:
`if (!EQ::ValueWithin(Class::Warrior, Class::Berserker))`

Reply:
> Drop `Client::HandleNameApprovalPacket` as it comes with logging for free now

Not sure what you mean here